src/php/ext/grpc/byte_buffer.c

```diff
@@ -18,6 +18,8 @@
 
 #include <php.h>
 
+// The include file must place here under <php.h> for fixing compile error.
+// See: https://github.com/grpc/grpc/pull/12360#issuecomment-326484589
```

Comment:
Interesting. I don't see anything inside `byte_buffer.h` requires `<php.h>`, can you please help me understand why this order matters?

Reply:
As https://github.com/grpc/grpc/pull/12360#issuecomment-326484589 mention errors, `_GNU_SOURCE` have defined at `support/port_platform.h` file, and `_GNU_SOURCE` also have defined `main/php_config.h`, maybe we should do something at `support/port_platform.h`. So we can use [it](https://github.com/grpc/grpc/pull/12360#issuecomment-326713716) fix the problem, but not beautiful.